app/env.yml

```diff
@@ -25,6 +25,5 @@ dependencies:
   - psutil==5.4.3
   - requests==2.18.4
   - srtm.py==0.3.2
-  - uplog==0.1
```

Comment:
@samwaterbury: I'm not familiar with python packages, so this may be a dumb question ---- 
should `logger` be added as a dependency here?

Reply:
Not in this instance since `logging` is a built-in module of Python: https://docs.python.org/3/library/logging.html